### PR TITLE
Adding danish and dutch

### PR DESCRIPTION
**Pull request by JoseSenart**
Hi again @jama5262 

I'm adding 2 more languages Dutch and Danish

### CODE REVIEW

File: lib/src/relative_time/locales/nl_locale.dart [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  @override
+  String aboutAMinute(int minutes) => 'een minuut';
+  @override
+  String minutes(int minutes) => '$minutes minuten';
+  @override
+  String aboutAnHour(int minutes) => 'een uur';
+  @override
+  String hours(int hours) => '$hours uur';
+  @override
+  String aDay(int hours) => 'a dag';
```
Comment (AFAS-HHO) [inline lib/src/relative_time/locales/nl_locale.dart:24]:
```suggestion
  String aDay(int hours) => 'een dag';
```

Comment (JoseSenart) [reply lib/src/relative_time/locales/nl_locale.dart:24]: indeed, thanks!

**Comment by WichardRiezebos** [comment]
Is it possible to merge?

**Comment by jama5262** [comment]
@JoseSenart Hi, would you please send this PR to the `develop` branch not the `master` branch

**Comment by jama5262** [comment]
@JoseSenart Thanks for the PR, but a lot is happening here same as #74 and some changes are not directly related to just adding the adding danish and dutch

For now, I will close this PR and request you to open other ones that specify their purpose

**Pull request closed without merging**